jetty9-compat-base/src/test/java/com/google/apphosting/vmruntime/jetty9/LoggingIT.java

```diff
@@ -71,12 +74,12 @@ public void testGet() throws Exception {
       data = new Gson().fromJson(line, JsonData.class);
       assertThat(data.severity, equalTo("ERROR"));
       assertThat(
-          data.message, org.hamcrest.Matchers.containsString("LoggingServlet doGet: not null"));
+          data.message, org.hamcrest.Matchers.containsString("com.foo.bar: not null"));
```

Comment:
Very good question.  I don't actually know how that test was passing before because the logs are explicitly generated from a "com.foo.bar" logger.

So I'll take this question on notice and research how it was working prior to this PR.